docs/src/security/waf.md

```diff
@@ -102,7 +102,7 @@ Platform.sh's WAF also includes several protective rules for vulnerabilities fou
 
 ### Magento
 
-#### [Magestore extension vulnerability (2019-SQLi)](https://magento.com/security/news/critical-vulnerability-magestore-store-locator-extension) 
+#### [Magestore extension vulnerability (2019-SQLi)](https://web.archive.org/web/20210414043507/https://magento.com/security/news/critical-vulnerability-magestore-store-locator-extensionstore-store-locator-extension) 
```

Comment:
I thought about it, but I decided figuring out whether it would be relevant would require time. It'd mean investigating whether people are likely to be using such a version. Given that people are still using PHP 5.4 and such (which reached end of life 8 years ago), I thought it'd be possible. And also discussing a policy for the page going forward. So I thought fixing the link would be enough for now.